modules/@apostrophecms/image/ui/apos/apps/AposImageRelationshipQueryFilter.js

```diff
@@ -0,0 +1,13 @@
+export default () => {
+  const queryOptions = [ 'minSize' ];
+
+  apos.bus.$on('piece-relationship-query', (query) => {
+    const [ options = {} ] = apos.area?.widgetOptions || [];
+
+    Object.entries(options).forEach(([ optName, optValue ]) => {
```

Comment:
Modified the code to parse only the `queryOptions` array, this way we'll be able to add more params by just adding their name into the array. Since this event if not specific to minSize, I would like to keep it the more generic.